website/docs/about/introduction/glossary.md

```diff
@@ -152,6 +152,10 @@ Refers to a [terraform module](https://www.terraform.io/docs/configuration/modul
 
 
 ## N
+### NGAP
+NASA General Application Platform. NGAP provides a cloud-based
+Platform-as-a-Service (PaaS) and Infrastructure-as-a-Service (IaaS) for
+ESDIS applications.
```

Comment:
Not sure if it matters or if we should add that it also provides our ATO to operate in the AWS commercial cloud.

Reply:
Added content into the NGAP definition for the ATO.